### PR TITLE
Add InventoryLevel and Variants all

### PR DESCRIPTION
**Pull request by dainiauskas**
Added https://shopify.dev/docs/admin-api/rest/reference/inventory/inventorylevel support

And Shopify undocumented:
/admin/variant.json to get all variants
/admin/variants/count.json to count all variants

### CODE REVIEW

File: inventory_level.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+// InventoryLevelServiceOp is the default implementation of the InventoryLevelService interface
+type InventoryLevelServiceOp struct {
+	client *Client
+}
+
+// InventoryLevel represents a Shopify inventory level
+type InventoryLevel struct {
+	InventoryItemID   int64      `json:"inventory_item_id,omitempty"`
+	LocationID        int64      `json:"location_id,omitempty"`
+	Available         int        `json:"available,omitempty"`
```
Comment (XciD) [inline inventory_level.go:30]: You should remove `omitempty`, in order to set the stock to 0, otherwise the json marshaller will omit the value.

Comment (dainiauskas) [reply inventory_level.go:30]: Sorry, about which line you talk? Can not find :(

Comment (XciD) [reply inventory_level.go:30]: If you create a `InventoryLevel` with `Available` set to 0, you will generate a json without the `available` property that will result to a 400 error.
You just need to remove `omitempty`

**Comment by gordcurrie** [comment]
Closing due to inactivity, please reopen when opportunity permits.

**Pull request closed without merging**